partcad/src/partcad/part_factory_obj.py

```
@@ -0,0 +1,152 @@
+
+            # Update counters
+            with PartFactoryObj.lock:
+                if do_subprocess:
+                    PartFactoryObj.count_inflight_subprocess -= 1
+                else:
+                    PartFactoryObj.count_inflight_simple -= 1
+
+            self.ctx.stats_parts_instantiated += 1
+            return shape
+
+    def _should_use_subprocess(self, file_size):
```

Ruff (0.8.2)

Missing return type annotation for private function `_should_use_subprocess`

Add return type annotation: `bool`

(ANN202)